fly.toml

```diff
@@ -0,0 +1,20 @@
+app = "microbin"
+primary_region = "ams"
+kill_signal = "SIGINT"
+kill_timeout = "5s"
+
+[experimental]
+  entrypoint = ["microbin", "--highlightsyntax", "--private", "--qr", "--editable", "--enable-burn-after"]
```

Comment:
Can users easily change these parameters?

Reply:
Yes, they can able to change the parameters.